data/remnant/remnant 1 introduction.txt

```diff
@@ -3821,12 +3821,29 @@ mission "Remnant: Logistics 1"
 	to offer
 		has "remnant: blood test pure"
 		has "Remnant: Defense 1: done"
+		has "Remnant: Technology Available: offered"
```

Comment:
The Technology Available mission requires all the missions in the Defense chain, and so we don't also need to check for that.
```suggestion
		has "Remnant: Technology Available: offered"
```